README.md

```diff
@@ -1,3 +1,15 @@
+## Getting started
+
+1. clone repo
+
+2. install dependencies
```

Comment:
nit: in the future, we can create a page for the dependencies and step by step start off the project